scientific_library/contrib/tvb/contrib/scripts/utils/time_series_utils.py

```diff
@@ -0,0 +1,219 @@
+# coding=utf-8
```

Comment:
Why these functions are not methods on the TS_Service class?

Reply:
To shorten ts service and to have the available for other possible uses.

Reply:
I see.... 
Just gathering a lot of functions under "utils" is not a good practice in programming, as it tends to grow without much structure (other that it is "useful" code for "some purpose", purpose which often remains unclear to the end user of that code).
We could leve these be, for the moment, but I advice to convert these in Services, where possible.

Reply:
Then, Bogdan should feel free to refactor at will inside contrib.
If I ever need to use these functions in another context, I can always copy-paste them from my own tvb-scripts repo.

Reply:
The same goes for file_utils for instance and all other utils.
Functions not used could be removed, and maybe re-added at another moment in future etc

Reply:
Good. 
Let us do this after we merge the current PR, as it already gathering many changes.